pkg/spdk/Dockerfile

```diff
@@ -0,0 +1,137 @@
+#Build SPDK
+FROM alpine:3.12 as build
+RUN apk add --no-cache linux-headers=5.4.5-r1 \
+                       bash=5.0.17-r0     \
+                       gcc=9.3.0-r2 \
+                       g++=9.3.0-r2 \
+                       nasm=2.14.02-r0 \
+                       autoconf=2.69-r2 \
+                       automake=1.16.2-r0 \
+                       libtool=2.4.6-r7 \
+                       gawk=5.1.0-r0 \
+                       meson=0.54.2-r1 \
+                       libc-dev=0.7.2-r3 \
+                       bsd-compat-headers=0.7.2-r3 \
+                       patch=2.7.6-r6 \
+                       argp-standalone=1.3-r4 \
+                       libexecinfo-dev=1.1-r1 \
+                       util-linux-dev=2.35.2-r0 \
+                       libressl-dev=3.1.2-r0 \
+                       libaio-dev=0.3.112-r1 \
+                       ncurses-dev=6.2_p20200523-r0 \
+                       cunit-dev=2.1.3-r2 \
+                       glib-dev=2.64.6-r0 \
+                       libusb-dev=1.0.23-r0 \
+                       pixman-dev=0.40.0-r2 \
+                       yajl-dev=2.1.0-r1 \
+                       libcap-ng-dev=0.7.10-r1 \
+                       attr-dev=2.4.48-r0 \
+                       file=5.38-r0 \
+                       git=2.26.2-r0 \
+                       make=4.3-r0
+
+ENV CC gcc
+COPY *patch /
+
+RUN mkdir -p /out
+
+ENV LIBURING_VERSION 0.7
+ENV SPDK_VER 20.10
+ENV DPDK_VER 20.11
+ENV ISAL_VER 2.30.0
+ENV QEMU_VER 5.2.0
+
+ENV LIBURING_SOURCE=https://git.kernel.dk/cgit/liburing/snapshot/liburing-${LIBURING_VERSION}.tar.bz2
+
+# Download and verify liburing
+RUN wget https://git.kernel.dk/cgit/liburing/snapshot/liburing-${LIBURING_VERSION}.tar.bz2
+RUN tar xvf /liburing-${LIBURING_VERSION}.tar.bz2
+
+WORKDIR /liburing-${LIBURING_VERSION}
+
+RUN ./configure --prefix=/usr
+RUN make src && make install && make install DESTDIR=/out
+
+WORKDIR /
+RUN git clone --depth 1 https://github.com/spdk/spdk.git --branch v${SPDK_VER}
+WORKDIR /spdk
+RUN git submodule update --init
+
+WORKDIR /
+RUN wget https://download.qemu.org/qemu-${QEMU_VER}.tar.xz 
+RUN tar xvf qemu-${QEMU_VER}.tar.xz
+
+WORKDIR /spdk/dpdk
```

Comment:
Can we please get rid of dpdk? There's really no reason we should build it -- and definitely no reason we should include it in the final package.